instrumentation/elastic-opentelemetry-instrumentation-openai/tests/test_embeddings.py

```diff
@@ -117,7 +117,7 @@ def test_all_the_client_options(provider_str, model, input_tokens, duration, tra
         GEN_AI_REQUEST_MODEL: model,
         GEN_AI_SYSTEM: "openai",
         GEN_AI_RESPONSE_MODEL: model,
-        GEN_AI_REQUEST_ENCODING_FORMAT: "float",
+        GEN_AI_REQUEST_ENCODING_FORMATS: ("float",),
```

Comment:
Possibly showing my ignorance here. The code sets value of this attribute to a *list*, but compares it to a *tuple* here. Why does the `==` comparison return True?

```python
>>> {"foo":1, "bar":["float"]} == {"foo":1, "bar":("float",)}
False
```

Reply:
Arrays ends up as tuples for some reason in tests, haven't digged much on where this happen.